loader/db.go

```diff
@@ -87,9 +88,11 @@ func (conn *DBConn) querySQL(ctx *tcontext.Context, query string, args ...interf
 					return ret, ret.Err()
 				}
 				cost := time.Since(startTime)
-				queryHistogram.WithLabelValues(conn.cfg.Name, conn.cfg.SourceID).Observe(cost.Seconds())
-				if cost.Seconds() > 1 {
-					ctx.L().Warn("query statement",
+				// duration seconds
+				ds := cost.Seconds()
+				queryHistogram.WithLabelValues(conn.cfg.Name, conn.cfg.SourceID).Observe(ds)
+				if ds > 1 {
+					ctx.L().Warn(fmt.Sprintf("query statement too slow for %vs", ds),
```

Comment:
zap field can be used to print the message, no need to introduce `fmt` package.

in fact `zap.Duration("cost time", cost)` has printed the duration.

Reply:
OK, I see.
But I think it would be more intuitive to put duration directly in message? Because there is a lot of information in the log, it is not easy to find the `cost time` field.
Or put `cost time`  in front of `query`?

Reply:
I prefer put cost time the first field, in front of query.

Reply:
Well, I'll revise it again.